dist/index.html

```diff
@@ -142,22 +142,21 @@ <h3>Sarra Miller</h3>
 
     </div>
   </section>
-
-  <!-- Promotions Section -->
-  <section class="promotions-section">
-    <div class="promotions-section__wrapper">
-      <div class="promotions-section__promotion">
-        <h2>try for free</h2>
-        <h3>free 5 days pass</h3>
-        <button class="btn btn-primary">get in touch</button>
-      </div>
-      <div class="promotions-section__contact">
-        <h2>call us and try</h2>
-        <h3>957 547 4545</h3>
-        <button class="btn btn-primary">get in touch</button>
-      </div>
-    </div>
-  </section>
+  <div class="promotion-contact-flexwrapper">
+    <!-- Promotions Section -->
+    <section class="promotions-section">
+      <h2>try for free</h2>
+      <p>free 5 days pass</p>
+      <button class="btn btn-primary">get in touch</button>
+    </section>
+    
+    <!-- Contact us Section -->
+    <section class="contact-section">
+      <h2>call us and try</h2>
+      <p>957 547 4545</p>
+      <button class="btn btn-primary">get in touch</button>
```

Comment:
when naming your CSS classes, try to not use generic names like `btn` unless the styling is truly generic, otherwise try to be more specific in the naming, an example from your code would be : `promotion-contact-flexwrapper`, you didn't jsut call it `flexwrapper`

Reply:
I actually used the generic styles for buttons made by @ahmedsamir96m, It's the same button across the page.